app/controllers/api/v1/api_controller.rb

```diff
@@ -6,6 +6,12 @@ class ApiController < ApplicationController
       protect_from_forgery with: :null_session
       before_action :authenticate_user!
       before_action :set_locale
+      rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
+
+      def user_not_authorized
+        flash[:warning] = "You have met a terrible fate, haven't you?"
+        redirect_to '/api/v1/book'
```

Comment:
Stange URL to redirect, but ok!

Reply:
There is other syntaxis from rails to url and path

Reply:
I don't have a root so I had to think of something else to redirect to-